### PR TITLE
Use correct module path

### PR DESCRIPTION
**Pull request by okuta**
rel #3364

### CODE REVIEW

File: tests/chainer_tests/functions_tests/loss_tests/test_sigmoid_cross_entropy.py [hunk 1]
```diff
@@ -206,7 +208,7 @@ def forward(self):
     def test_call_cudnn_backward(self):
         with chainer.using_config('use_cudnn', self.use_cudnn):
             y = self.forward()
-            patch = 'cupy.cuda.cudnn.activationForward_v4'
+            patch = 'cupy.cudnn.activation_forward'
```
Comment (niboshi) [inline tests/chainer_tests/functions_tests/loss_tests/test_sigmoid_cross_entropy.py]: Why is this change needed?

File: tests/chainer_tests/functions_tests/loss_tests/test_sigmoid_cross_entropy.py [hunk 2]
```diff
@@ -185,7 +185,9 @@ def test_backward_no_reduction_gpu_no_cudnn(self):
 
 
 @testing.parameterize(
-    {'use_cudnn': 'always'},
+    # TODO(okuta) : fix
+    # TypeError: Unsupported type <class 'mock.mock.MagicMock'>
+    # {'use_cudnn': 'always'},
```
Comment (niboshi) [inline tests/chainer_tests/functions_tests/loss_tests/test_sigmoid_cross_entropy.py]: What causes this error?

File: tests/chainer_tests/functions_tests/loss_tests/test_sigmoid_cross_entropy.py (hunk 1 above)
Comment (okuta) [reply tests/chainer_tests/functions_tests/loss_tests/test_sigmoid_cross_entropy.py]: Cython behinds `activationForward_v4` call .

File: tests/chainer_tests/functions_tests/loss_tests/test_sigmoid_cross_entropy.py (hunk 2 above)
Comment (okuta) [reply tests/chainer_tests/functions_tests/loss_tests/test_sigmoid_cross_entropy.py]: `mock` returns `mock.mock.MagicMock` type. So, `isinstance` is failed.

Comment (niboshi) [reply tests/chainer_tests/functions_tests/loss_tests/test_sigmoid_cross_entropy.py]: How about:
```
            with mock.patch.object(
                    cuda.cupy.cudnn, 'activation_forward',
                    wraps=cuda.cupy.cudnn.activation_forward) as func:
```

File: tests/chainer_tests/functions_tests/loss_tests/test_sigmoid_cross_entropy.py (hunk 1 above)
Comment (niboshi) [reply tests/chainer_tests/functions_tests/loss_tests/test_sigmoid_cross_entropy.py]: OK.

File: tests/chainer_tests/functions_tests/loss_tests/test_sigmoid_cross_entropy.py (hunk 2 above)
Comment (okuta) [reply tests/chainer_tests/functions_tests/loss_tests/test_sigmoid_cross_entropy.py]: I will try

**Comment by niboshi** [comment]
LGTM!
